src/screen/PushSettingsScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {Divider, Icon, Layout, ListItem, Text} from '@ui-kitten/components';
+import GenericNavigationLayout from 'presentational/GenericNavigationLayout';
+import {DrawerNavigationProp} from '@react-navigation/drawer';
+import {StyleSheet, Switch, View} from 'react-native';
+import globalStyles, {standardPadding} from 'src/styles';
+import Padder from 'presentational/Padder';
+import {useDataContext} from 'context/DataContext';
+import {useMutation, useQuery, useQueryClient} from 'react-query';
+import LoadingView from 'presentational/LoadingView';
+import messaging from '@react-native-firebase/messaging';
+
+type PropTypes = {
+  navigation: DrawerNavigationProp<DrawerParamList>;
+};
+
+export function PushSettingsScreen(props: PropTypes) {
+  const {navigation} = props;
+  const {topics, toggleTopic, isLoading} = useTopics();
+
+  return (
+    <GenericNavigationLayout title="Push Notifications" onBackPressed={() => navigation.goBack()}>
+      <Layout level="1" style={styles.container}>
+        <Text>Hi there! To stay informed choose which Push Notifications you'd like to receive.</Text>
+        <Padder scale={2} />
+        <Divider />
+        <Padder scale={1} />
+        <View style={globalStyles.flex}>
+          {isLoading ? (
+            <LoadingView />
+          ) : (
+            topics.map((item) => (
+              <ListItem
+                key={item.id}
+                title={item.label}
+                accessoryRight={() => (
+                  <Switch value={item.selected} onValueChange={(subscribe) => toggleTopic({id: item.id, subscribe})} />
+                )}
+              />
+            ))
+          )}
+          <Padder scale={2} />
+          <Divider />
+        </View>
+        <ListItem
+          accessoryLeft={(p) => <Icon {...p} name={'info-outline'} />}
+          title={"Don't forget to enable notifications in your phone setting in the app section"}
+        />
+      </Layout>
+    </GenericNavigationLayout>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: standardPadding * 3,
+  },
+});
+
+const TOPICS = [
+  {id: 'REF_VOTE', label: 'Vote on Active Referenda'},
+  {id: 'COUNCIL_VOTE', label: 'Vote for Council Election'},
+  {id: 'EMERGENCY_VOTE', label: 'Vote for Emergency Proposals'},
+  {id: 'TREASURY_VOTE', label: 'Vote for Treasury Proposals'},
```

Comment:
We probably should use the chain event names as ids so we don't have to map them